src/serac/infrastructure/accelerator.hpp

```diff
@@ -85,21 +87,53 @@ void initializeDevice();
 void terminateDevice();
 
 #if defined(__CUDACC__)
+
 /**
  * @brief Utility method to display last cuda error message
  *
- * @param[in] o The output stream to post success or CUDA error messages
  * @param[in] success_string A string to print if there are no CUDA error messages
+ * @param[in] exit_on_error Exit on CUDA error
  */
-inline void displayLastCUDAErrorMessage(std::ostream& o, const char* success_string = "")
+inline void displayLastCUDAErrorMessage(const char* success_string = "", bool exit_on_error = false)
```

Comment:
This doesn't display only errors at this point.  Should be changed to `displayLastCUDAMessage`.